_episodes/01-welcome.md

```diff
@@ -110,7 +110,7 @@ organization before starting the training.
 
 Software Carpentry, Data Carpentry, and Library Carpentry
 are official Lesson Programs of The Carpentries.
-Together, they form a **global community of volunteer researchers, educators, and others** oriented around
+Around them, The Carpentries forms a **global community of volunteer researchers, educators, and others** oriented around
```

Comment:
```suggestion
The Carpentries is a **global community of volunteer researchers, educators, and others** oriented around
```

The original sentence sort of had historical baggage that doesn't need to be here. We expand on this later, so I think an overall simpler sentence is more clear here.

Reply:
I adopted your suggestion. Thanks